.github/workflows/ci.yml

```
@@ -6,39 +6,9 @@ concurrency:
-      - 'tests/**'
-      - 'infra/**'
-      - 'scripts/**'
-      - 'docker/**'
-      - 'docker-compose*.yml'
-      - 'pyproject.toml'
-      - 'uv.lock'
-      - 'go.mod'
-      - 'go.sum'
-      - 'package.json'
-      - 'bun.lock'
+    branches: [ main, master ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)